content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Quality Gates
+kind: documentation
+description: Learn how to use Quality Gates in your pipeline.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/tests"
+  tag: "Documentation"
+  text: "Learn about Test Visibility"
+---
+
+## Overview
+
+{{< callout url="#" header="false" >}}
+Quality Gates Rules are in beta.
+{{< /callout >}}
+
+Quality Gates allows you to gate your workflows based on signals in Datadog.
+For example, you can block a Pull Request from being merged if it introduces new flaky tests, even if the tests are passing.
+
+Using Quality Gates, you have control over what is merged into the default branch, and ultimately on what is deployed. ([AM] ADD SENTENCE)
+
+## Set up Quality Gates
+
+There are two main steps required to set up Quality Gates:
+1. Create one or more rules.
+2. Add the `datadog-ci gate evaluate` command in your CI pipeline.
+
+## Create a rule
+
+To create Quality Gates rules for your organization, your user account must have the `quality_gate_rules_write` [permission][1].
+
+1. In Datadog, navigate to [**CI** > **Quality Gate Rules**][2] and click **+ New Rule**.
+2. Select the rule type. You can choose between `Static Analysis` and `Test`.
+3. Define the rule scope. The rule scope defines when the rules should be evaluated. For example, you can scope a rule so that
+it is evaluated only on specific repositories and branches. To define the scope for a rule, switch to "select when to evaluate" and add included
+or excluded values for the scope name. More information on rule scopes ([AM] ADD LINK TO SECTION BELOW).
+4. Define the rule condition. The rule condition states in which scenario the rule fails, failing the related pipeline as well.
+You can select one of the existing rule conditions for the rule type you have selected.
+The following example shows how to create a static analysis rule that will fail when there are one or more static analysis
+violations with "error" severity and "security" category being introduced in a specific commit:
+{{< img src="ci/qg_rule_condition_sa_errors_security.png" alt="Rule for static analysis security errors" style="width:80%;">}}
+5. Select whether the rule should block the pipeline or not when it fails. Non-blocking rules are still evaluated, but
+they do not block the pipeline when they fail.
```

Comment:
```suggestion
5. Select whether or not the rule should block the pipeline when it fails. Non-blocking rules are still evaluated, but they do not block the pipeline when they fail.
```

Reply:
The suggestion sounds weird to me:

Non-blocking rules are evaluated **still**, they do not block the pipeline when they fail. ?